src/use_cases_execution/schedules.rb

```diff
@@ -69,5 +69,10 @@ def self.load
       { path: "#{__dir__}/save_backup/save_backup_in_r2.rb", time: ['00:00'] },
       { path: "#{__dir__}/save_backup/delete_older_backup_in_r2.rb", time: ['00:20'] }
     ].freeze
+
+    MISSING_WORK_LOGS_SCHEDULES = [
```

Comment:
Just to clarify, they will run daily at those times correct?

Reply:
Hello @L-Zuluaga, yes, the BOT will run every day at 8:30 AM (BOG)